manifests/server/config.pp

```diff
@@ -41,6 +41,7 @@
   $rest            = $mongodb::server::rest
   $slowms          = $mongodb::server::slowms
   $keyfile         = $mongodb::server::keyfile
+  $key             = $mongodb::server::key
```

Comment:
Unfortunately, I am backlogged on the project I'm currently working and do not have time to work on this PR. Let me know if I should close it.